app/javascript/components/enrollment/steps/ExposureInformation.js

```diff
@@ -427,25 +494,47 @@ class ExposureInformation extends React.Component {
               type="switch"
               id="member_of_a_common_exposure_cohort"
               label="MEMBER OF A COMMON EXPOSURE COHORT"
-              checked={this.state.current.patient.member_of_a_common_exposure_cohort || false}
-              onChange={this.handleChange}
-            />
-          </Form.Group>
-          <Form.Group as={Col} md="auto" className="mb-0 my-auto ml-4">
-            <Form.Control
-              size="sm"
-              className="form-square"
-              id="member_of_a_common_exposure_cohort_type"
-              placeholder="enter description"
-              value={this.state.current.patient.member_of_a_common_exposure_cohort_type || ''}
-              onChange={this.handleChange}
-              aria-label="Enter Cohort Description"
+              checked={this.state.current.common_exposure_cohorts?.length > 0}
+              disabled={true}
             />
-            <Form.Control.Feedback className="d-block" type="invalid">
-              {this.state.errors['member_of_a_common_exposure_cohort_type']}
-            </Form.Control.Feedback>
           </Form.Group>
         </Form.Row>
+        {this.state.current.common_exposure_cohorts?.length > 0 && (
+          <div className="common-exposure-cohort-table-wrapper">
+            <CommonExposureCohortsTable
+              common_exposure_cohorts={this.state.current.common_exposure_cohorts}
+              isEditable={true}
+              onEditCohort={index => this.toggleCommonExposureCohortModal(true, this.state.current.common_exposure_cohorts[`${index}`], index)}
+              onDeleteCohort={this.handleCohortDelete}
+            />
+          </div>
+        )}
+        {this.state.current.common_exposure_cohorts?.length >= 10 ? (
```

Comment:
Two things here...

1) I'm seeing the demo script populate monitorees with way over this limit (see screenshot).  if the true limit is 10, this should be prevented
<img width="713" alt="Screen Shot 2021-11-12 at 3 49 59 PM" src="https://user-images.githubusercontent.com/35042815/141533768-7fd093f4-2d67-4053-baa7-37dd9007e99e.png">

2) You shouldn't need ternary for this.  Have one button and use `this.state.current.common_exposure_cohorts?.length >= 10` to show the tooltip and disable the button

Reply:
I'm also not seeing this tooltip work so I would use `ReactTooltip`

Reply:
2. It actually doesn't work like that unfortunately

Reply:
it's not prevented in the demo script because the way the cohorts are created are via bulk import which doesn't run any validations, but I will fix it

Reply:
fixed demo script here: https://github.com/SaraAlert/SaraAlert/pull/1125/commits/eff7705eebef91c92d3528cf9dcba2e1dd7cd22c#diff-9bfc38ef71b77112eb91f77e35668d17dfea639c79e19de03900b201ad44b8efR269

Reply:
For #2 that should work if you use `ReactTooltip`.  I'm still not seeing the tooltip render as is

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/9a3c80cdc1991ea818b06bb99362ad971f4875d1#